src/box/box.cc

```diff
@@ -6225,6 +6227,7 @@ box_free(void)
 	tuple_free();
 	port_free();
 	iproto_constants_free();
+	rt_access_destroy();
```

Comment:
The order of the deinitialization is generally the reverse of the initialization order. It is not so in the given case. If there is a reason, why it should be placed here, please comment. Otherwise, please, adjust the order.

Reply:
Order changed.